src/react/workflow/Workflows.tsx

```diff
@@ -87,7 +104,7 @@ export default function Workflows() {
   );
 
   const select = (workflows: APIWorkflows) => makeWorkflows(workflows);
-  const workflowListDataQuery = useWorkflows(select);
+  const workflowListDataQuery = useWorkflowsWithSelect(select);
```

Comment:
nit: select can be inlined here
```suggestion
  const workflowListDataQuery = useWorkflowsWithSelect(makeWorkflows);
```